File: app/src/main/java/org/zeroxlab/imgorg/SettingsActivity.java
// vim: et sw=4 sts=4 tabstop=4
package org.zeroxlab.imgorg;

import android.app.Activity;
import android.os.Bundle;


public class SettingsActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        // Display the fragment as the main content.
        getFragmentManager().beginTransaction()
                .replace(android.R.id.content, new SettingsFrag())
                .commit();
    }
}

File: app/src/main/java/cc/jchu/imgorg/SettingsActivity.kt
// vim: et sw=4 sts=4 tabstop=4
package cc.jchu.imgorg

import android.R
import android.app.Activity
import android.os.Bundle

class SettingsActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        // Display the fragment as the main content.
        fragmentManager.beginTransaction()
            .replace(R.id.content, SettingsFragment())
            .commit()
    }
}
